Automattic/simplenote-electron
# Add/email verification

### Fix

Adds the Email Verification UI to either update or send verification email and dialog reminding users to click the verification link that is in the email we sent them.

### Test
1. Have an unverified account
2. Log in to the app
3. Click the button to change email
4. Change email
5. Refresh app
6. Click button to send email
7. Observe that dialog changes to the secondary dialog asking you to check your email
8. Click link in email
9. Observe dialog disappears
10. Log out
11. Log in
12. Refresh
13. Observe dialog does not appear again

### Release

- Added email verification UI
